kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/AdminResource.scala

```diff
@@ -246,7 +246,12 @@ private[v1] class AdminResource extends ApiRequestContext with Logging {
       @QueryParam("sharelevel") shareLevel: String,
       @QueryParam("subdomain") subdomain: String,
       @QueryParam("proxyUser") kyuubiProxyUser: String,
-      @QueryParam("hive.server2.proxy.user") hs2ProxyUser: String): Response = {
+      @QueryParam("hive.server2.proxy.user") hs2ProxyUser: String,
+      @QueryParam("forceKill") @DefaultValue("false") forceKill: Boolean,
+      @QueryParam("refId") refId: String,
```

Comment:
refId is not needed, we can get it with user and specified `sharelevel`, `subdomain`.

Reply:
otherwise, Kyuubiserver can not check whether you have permission to kill the engine.

You can specify any refId to kill, it does not make senses.

Reply:
See DiscoveryClient::getServiceNodesInfo

Reply:
OK, that makes sense. I can get refId by below listEngines method, right?